Week3/homework/src/dump.sql

```diff
@@ -0,0 +1,105 @@
+-- MySQL dump 10.17  Distrib 10.3.13-MariaDB, for Linux (x86_64)
+--
+-- Host: localhost    Database: todo
+-- ------------------------------------------------------
+-- Server version	10.3.13-MariaDB
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8mb4 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `items`
+--
+
+DROP TABLE IF EXISTS `items`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `items` (
+  `item_id` int(11) NOT NULL AUTO_INCREMENT,
+  `list_id` int(11) NOT NULL,
+  `description` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `completion` tinyint(1) NOT NULL DEFAULT 0,
+  PRIMARY KEY (`item_id`),
+  KEY `fk2` (`list_id`),
```

Comment:
**Non-blocking:**

It would be a good thing to name foreign keys properly.

**Saying this because of** 
- when you see those keys on the diagram, and when there a lot of them - you can easily get lost without names
- then FK fails - it would be great to see it's name in the error message, it immediately gives you a nice clue about what went wrong

Reply:
Foreign key names are changed with meaningful ones.